modules/tinymce/src/core/test/ts/browser/html/NodeTest.ts

```diff
@@ -310,7 +284,7 @@ UnitTest.asynctest('browser.tinymce.core.html.NodeTest', function (success, fail
   });
 
   suite.test('clone', function () {
-    let root, node, clone;
+    let  node, clone;
```

Comment:
```ts
let  node
```

or

```ts
let node
```

Definitely a minor nit but also this seems like the PR for minor nits?

Reply:
Yeah, that's a bug. Thanks for the catch! I wonder why eslint didn't catch this...